stock_available_to_promise_release/models/stock_move.py

```diff
@@ -363,6 +389,42 @@ def _prepare_move_split_vals(self, qty):
             vals.update({"procure_method": self.procure_method, "need_release": True})
         return vals
 
+    def _get_release_decimal_precision(self):
+        return self.env["decimal.precision"].precision_get("Product Unit of Measure")
+
+    def _get_release_remaining_qty(self):
+        self.ensure_one()
+        quantity = min(self.product_qty, self.ordered_available_to_promise_qty)
+        remaining = self.product_qty - quantity
+        precision = self._get_release_decimal_precision()
+        if not float_compare(remaining, 0, precision_digits=precision) > 0:
+            return
+        return remaining
+
+    def _is_releasable(self):
```

Comment:
IMO, the method name should be changed to something more meaningful. When you read the code, you don't expect that a tuple will be returned as result to a call to the method. A docstring would be a plus.

Reply:
Yes i think so too, i will create a PR for this.

Reply:
now included here